src/main/scala/devices/debug/dm_registers.scala

```diff
@@ -1,8 +1,8 @@
-package freechips.rocketchip.devices.debug
+package uncore.devices
```

Comment:
you have to fix this. Ah, now I see why you were importing `uncore.devices` earlier. We should fix the `make chisel` target to to do the right thing in `riscv-debug-spec`